lib/Cleantalk/ApbctWP/State.php

```diff
@@ -430,7 +431,8 @@ protected function init()
         // Standalone or main site
         $this->api_key        = $this->settings['apikey'];
         $this->dashboard_link = 'https://cleantalk.org/my/' . ($this->user_token ? '?user_token=' . $this->user_token : '');
-        $this->notice_show    = $this->data['notice_trial'] || $this->data['notice_renew'] || $this->data['notice_incompatibility'] || $this->isHaveErrors();
+        $this->notice_show    = $this->data['notice_trial'] || $this->data['notice_renew'] || $this->isHaveErrors();
+        $this->account_email  = $this->data['account_email'];
```

Comment:
This property ($apbct->account_email) is not used in the code. So this assignment is unnecessary.